.github/workflows/e2e_libvirt.yaml

```diff
@@ -105,6 +109,7 @@ jobs:
 
           echo "::group::cloud-api-adaptor logs"
           kubectl logs -l app=cloud-api-adaptor -n confidential-containers-system
+          kubectl logs ds/peerpodconfig-ctrl-caa-daemon -n confidential-containers-system
```

Comment:
could you put this in a new `::group::`?

Reply:
Updating now.